programming/vision/index.md

```diff
@@ -95,6 +84,31 @@ robot.camera.save(robot.usbkey / "photo.jpg", frame=frame)
 ~~~~~
 
 
+### Saving camera output
+
+You can also save a snapshot of what your webcam is currently seeing. This can be useful to debug your code.
+
+This is done by adding the `save` parameter to the `see` or `capture` functions.
+The parameter should be the filename to where you want to save the image.
+
+If used with the `see` function, every marker that your robot can see will have a square annotated around it, with a red dot indicating the top left corner of the marker.
+The ID of every marker is also written next to it.
+
+Snapshots are saved to your USB drive, and can be viewed on another computer.
+
+```python
+from sr.robot3 import *
+
+robot = Robot()
+
+# Save a snapshot of what the camera is currently seeing
+frame =  robot.camera.capture(save="snapshot.jpg")
```

Comment:
Suggestion: Is it worth adding a warning or something that images are overwritten, and potentially to use sensible naming to help understand which image is from when?